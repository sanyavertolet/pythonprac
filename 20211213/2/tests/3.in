[3157, -453, 3426, -441, -1897, -1963, -10000, 3332, 543, 5561, -7457, 5050, -7243, 8322, 5036, -5671, 6198, -1938, 9305, -188, 7233, 9620, -1814, -8960, -3294, -4785, 7912, -553, 4676, 6524, -8991, 3045, -6210, -9384, -2583, 9806, 2427, -3683, -493, 8143, -6436, -8597, 4662, -5157, 4766, 9647, -876, -4615, 4526, -3573, 2647, -6221, 5065, -4958, 7956, -9444, 2188, 764, 3903, -9201, 8348, 9814, 7786]
