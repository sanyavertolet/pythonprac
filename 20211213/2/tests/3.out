[-10000, -9444, -9384, -9201, -8991, -8960, -8597, -7457, -7243, -6436, -6221, -6210, -5671, -5157, -4958, -4785, -4615, -3683, -3573, -3294, -2583, -1963, -1938, -1897, -1814, -876, -553, -493, -453, -441, -188, 543, 764, 2188, 2427, 2647, 3045, 3157, 3332, 3426, 3903, 4526, 4662, 4676, 4766, 5036, 5050, 5065, 5561, 6198, 6524, 7233, 7786, 7912, 7956, 8143, 8322, 8348, 9305, 9620, 9647, 9806, 9814]
